test/framework/src/main/java/org/elasticsearch/test/InternalAggregationTestCase.java

```diff
@@ -317,13 +325,28 @@ public final void testFromXContent() throws IOException {
         } else {
             originalBytes = toXContent(aggregation, xContentType, params, humanReadable);
         }
+        BytesReference mutated;
+        if (addRandomFields) {
+            /**
+             * - we don't add to the root object because it should only contain the named aggregation to test
```

Comment:
ok, just to confirm, this is based on the assumption that in

```
"aggregations" : {
    "someType#xyz" : {
    },
   "someOtherType#abc" : {
    }
}
```

the only field that can be present within the aggregations object is an aggregation itself, nothing else. Right?